ERROR    - 2019-01-23 12:50:29,566 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object is not subscriptable
ERROR    - 2019-01-23 12:50:29,568 [db.py][insert_query]  Failed insert query: (1406, "Data too long for column 'court_name' at row 1")
ERROR    - 2019-01-23 12:53:13,687 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object is not subscriptable
ERROR    - 2019-01-23 12:56:11,872 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object is not subscriptable
ERROR    - 2019-01-23 12:56:53,967 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object is not subscriptable
ERROR    - 2019-01-23 12:57:59,979 [rulings_by_advance_authority.py][request_data]  Failed to request: request_pdf() missing 1 required positional argument: 'case_id'
ERROR    - 2019-01-23 13:00:15,047 [rulings_by_advance_authority.py][request_data]  Failed to request: must be str, not NoneType
ERROR    - 2019-01-23 13:01:01,971 [rulings_by_advance_authority.py][request_data]  Failed to request: must be str, not NoneType
ERROR    - 2019-01-23 13:02:11,768 [rulings_by_advance_authority.py][request_data]  Failed to request: must be str, not NoneType
ERROR    - 2019-01-23 13:03:09,026 [rulings_by_advance_authority.py][request_data]  Failed to request: unsupported operand type(s) for +: 'NoneType' and 'str'
ERROR    - 2019-01-23 13:03:59,600 [rulings_by_advance_authority.py][request_data]  Failed to request: must be str, not NoneType
ERROR    - 2019-01-23 13:04:24,522 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object is not subscriptable
ERROR    - 2019-01-23 13:06:22,562 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object is not subscriptable
ERROR    - 2019-01-23 13:06:52,504 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object is not subscriptable
ERROR    - 2019-01-23 13:07:34,853 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object is not subscriptable
ERROR    - 2019-01-23 13:08:46,353 [rulings_by_advance_authority.py][request_data]  Failed to request: must be str, not NoneType
ERROR    - 2019-01-23 13:09:08,169 [rulings_by_advance_authority.py][request_data]  Failed to request: must be str, not NoneType
ERROR    - 2019-01-23 13:09:24,936 [rulings_by_advance_authority.py][request_data]  Failed to request: must be str, not NoneType
ERROR    - 2019-01-23 13:10:30,577 [rulings_by_advance_authority.py][request_data]  Failed to request: must be str, not NoneType
ERROR    - 2019-01-23 13:12:15,067 [rulings_by_advance_authority.py][request_data]  Failed to request: request_pdf() missing 1 required positional argument: 'case_id'
ERROR    - 2019-01-23 13:16:01,290 [extract_text.py][pdf_to_text_api]  Failed to parse the html: No /Root object! - Is this really a PDF?
ERROR    - 2019-01-23 13:16:01,291 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:21:12,528 [extract_text.py][pdf_to_text_api]  Failed to parse the html: No /Root object! - Is this really a PDF?
ERROR    - 2019-01-23 13:21:12,528 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:21:45,778 [extract_text.py][pdf_to_text_api]  Failed to parse the html: No /Root object! - Is this really a PDF?
ERROR    - 2019-01-23 13:21:45,778 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:24:32,621 [extract_text.py][pdf_to_text_api]  Failed to parse the html: No /Root object! - Is this really a PDF?
ERROR    - 2019-01-23 13:24:32,622 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:25:06,935 [extract_text.py][pdf_to_text_api]  Failed to parse the html: No /Root object! - Is this really a PDF?
ERROR    - 2019-01-23 13:25:06,936 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:25:57,629 [extract_text.py][pdf_to_text_api]  Failed to parse the html: No /Root object! - Is this really a PDF?
ERROR    - 2019-01-23 13:25:57,629 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:26:14,624 [extract_text.py][pdf_to_text_api]  Failed to parse the html: No /Root object! - Is this really a PDF?
ERROR    - 2019-01-23 13:26:14,625 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:34:14,872 [extract_text.py][pdf_to_text_api]  Failed to parse the html: No /Root object! - Is this really a PDF?
ERROR    - 2019-01-23 13:34:14,873 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:36:31,175 [rulings_by_advance_authority.py][request_pdf]  Failed to get text file for: advance-authority-n-23-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:36:31,184 [extract_text.py][pdf_to_text_api]  Failed to parse the html: expected str, bytes or os.PathLike object, not NoneType
ERROR    - 2019-01-23 13:36:31,184 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:37:08,637 [rulings_by_advance_authority.py][request_pdf]  Failed to get text file for: advance-authority-n-23-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:37:08,659 [extract_text.py][pdf_to_text_api]  Failed to parse the html: expected str, bytes or os.PathLike object, not NoneType
ERROR    - 2019-01-23 13:37:08,660 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:39:41,311 [extract_text.py][pdf_to_text_api]  Failed to parse the html: No /Root object! - Is this really a PDF?
ERROR    - 2019-01-23 13:39:41,312 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:45:11,187 [db.py][insert_query]  Failed insert query: (1146, "Table 'new_courts_data.ruling_by_advance_authority' doesn't exist")
ERROR    - 2019-01-23 13:45:11,189 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:45:11,197 [db.py][update_query]  Failed update query: (1146, "Table 'new_courts_data.ruling_by_advance_authority' doesn't exist")
ERROR    - 2019-01-23 13:45:11,204 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:45:16,968 [db.py][insert_query]  Failed insert query: (1146, "Table 'new_courts_data.ruling_by_advance_authority' doesn't exist")
ERROR    - 2019-01-23 13:45:16,969 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:45:16,976 [db.py][update_query]  Failed update query: (1146, "Table 'new_courts_data.ruling_by_advance_authority' doesn't exist")
ERROR    - 2019-01-23 13:45:16,981 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:45:22,900 [db.py][insert_query]  Failed insert query: (1146, "Table 'new_courts_data.ruling_by_advance_authority' doesn't exist")
ERROR    - 2019-01-23 13:45:22,901 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:45:22,933 [db.py][update_query]  Failed update query: (1146, "Table 'new_courts_data.ruling_by_advance_authority' doesn't exist")
ERROR    - 2019-01-23 13:45:22,940 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:45:29,043 [db.py][insert_query]  Failed insert query: (1146, "Table 'new_courts_data.ruling_by_advance_authority' doesn't exist")
ERROR    - 2019-01-23 13:45:29,044 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:45:29,052 [db.py][update_query]  Failed update query: (1146, "Table 'new_courts_data.ruling_by_advance_authority' doesn't exist")
ERROR    - 2019-01-23 13:45:29,057 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:49:31,642 [extract_text.py][pdf_to_text_api]  Failed to parse the html: No /Root object! - Is this really a PDF?
ERROR    - 2019-01-23 13:49:31,643 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:51:05,688 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-23-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:51:05,704 [extract_text.py][pdf_to_text_api]  Failed to parse the html: expected str, bytes or os.PathLike object, not NoneType
ERROR    - 2019-01-23 13:51:05,704 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:52:09,586 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-23-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:52:09,600 [extract_text.py][pdf_to_text_api]  Failed to parse the html: expected str, bytes or os.PathLike object, not NoneType
ERROR    - 2019-01-23 13:52:09,602 [rulings_by_advance_authority.py][request_data]  Failed to request: 'NoneType' object has no attribute 'translate'
ERROR    - 2019-01-23 13:55:43,909 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-23-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:55:43,921 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            23/WBAAR/2018-19 
ERROR    - 2019-01-23 13:55:43,933 [rulings_by_advance_authority.py][request_data]  Failed to request: local variable 'pdf_text_data' referenced before assignment
ERROR    - 2019-01-23 13:56:34,802 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-23-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:56:34,833 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            23/WBAAR/2018-19 
ERROR    - 2019-01-23 13:56:34,848 [db.py][insert_query]  Failed insert query: (1366, "Incorrect integer value: '\\n            1          ' for column 's_no' at row 1")
ERROR    - 2019-01-23 13:56:34,849 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:56:37,771 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-22-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:56:37,787 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            22/WBAAR/2018-19 
ERROR    - 2019-01-23 13:56:37,801 [db.py][insert_query]  Failed insert query: (1366, "Incorrect integer value: '\\n            2          ' for column 's_no' at row 1")
ERROR    - 2019-01-23 13:56:37,805 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:56:41,458 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-21-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:56:41,488 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            21/WBAAR/2018-19 
ERROR    - 2019-01-23 13:56:41,503 [db.py][insert_query]  Failed insert query: (1366, "Incorrect integer value: '\\n            3          ' for column 's_no' at row 1")
ERROR    - 2019-01-23 13:56:41,506 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:56:44,734 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-20-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:56:44,757 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            20/WBAAR/2018-19 
ERROR    - 2019-01-23 13:56:44,773 [db.py][insert_query]  Failed insert query: (1366, "Incorrect integer value: '\\n            4          ' for column 's_no' at row 1")
ERROR    - 2019-01-23 13:56:44,776 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:56:48,318 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-tsaar-order-no-12-2018-09-18.pdf
ERROR    - 2019-01-23 13:56:48,348 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            TSAAR Order No. 12/2018 
ERROR    - 2019-01-23 13:56:48,362 [db.py][insert_query]  Failed insert query: (1366, "Incorrect integer value: '\\n            5          ' for column 's_no' at row 1")
ERROR    - 2019-01-23 13:56:48,366 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:56:52,114 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-raj-aar-2018-19-25-11-18.pdf
ERROR    - 2019-01-23 13:56:52,127 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            RAJ/AAR/2018-19/25 
ERROR    - 2019-01-23 13:56:52,142 [db.py][insert_query]  Failed insert query: (1366, "Incorrect integer value: '\\n            6          ' for column 's_no' at row 1")
ERROR    - 2019-01-23 13:56:52,145 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:56:58,455 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-raj-aar-2018-19-24-11-18.pdf
ERROR    - 2019-01-23 13:56:58,464 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            RAJ/AAR/2018-19/24 
ERROR    - 2019-01-23 13:56:58,471 [db.py][insert_query]  Failed insert query: (1366, "Incorrect integer value: '\\n            7          ' for column 's_no' at row 1")
ERROR    - 2019-01-23 13:56:58,472 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:57:33,143 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-23-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:57:33,172 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            23/WBAAR/2018-19 
ERROR    - 2019-01-23 13:57:33,188 [db.py][insert_query]  Failed insert query: (1406, "Data too long for column 'category_as_per_sgst_act' at row 1")
ERROR    - 2019-01-23 13:57:33,191 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:57:36,242 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-22-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:57:36,255 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            22/WBAAR/2018-19 
ERROR    - 2019-01-23 13:57:36,270 [db.py][insert_query]  Failed insert query: (1406, "Data too long for column 'category_as_per_sgst_act' at row 1")
ERROR    - 2019-01-23 13:57:36,273 [db.py][update_query]  Failed update query: not all arguments converted during string formatting
ERROR    - 2019-01-23 13:59:07,890 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-23-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:59:07,920 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            23/WBAAR/2018-19 
ERROR    - 2019-01-23 13:59:11,269 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-22-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:59:11,281 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            22/WBAAR/2018-19 
ERROR    - 2019-01-23 13:59:14,750 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-21-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:59:14,768 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            21/WBAAR/2018-19 
ERROR    - 2019-01-23 13:59:18,101 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-20-wbaar-2018-19-11-18.pdf
ERROR    - 2019-01-23 13:59:18,117 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            20/WBAAR/2018-19 
ERROR    - 2019-01-23 13:59:21,509 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-tsaar-order-no-12-2018-09-18.pdf
ERROR    - 2019-01-23 13:59:21,539 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            TSAAR Order No. 12/2018 
ERROR    - 2019-01-23 13:59:24,888 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-raj-aar-2018-19-25-11-18.pdf
ERROR    - 2019-01-23 13:59:24,904 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            RAJ/AAR/2018-19/25 
ERROR    - 2019-01-23 13:59:28,680 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-raj-aar-2018-19-24-11-18.pdf
ERROR    - 2019-01-23 13:59:28,693 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            RAJ/AAR/2018-19/24 
ERROR    - 2019-01-23 13:59:32,056 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-raj-aar-2018-19-23-11-18.pdf
ERROR    - 2019-01-23 13:59:32,087 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            RAJ/AAR/2018-19/23 
ERROR    - 2019-01-23 13:59:35,436 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-04-odisha-aar-18-19-10-18.pdf
ERROR    - 2019-01-23 13:59:35,448 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            04/Odisha/AAR/18-19 
ERROR    - 2019-01-23 13:59:38,780 [rulings_by_advance_authority.py][request_pdf]  Failed to get pdf file for: advance-authority-n-04-odisha-aar-18-19-10-18.pdf
ERROR    - 2019-01-23 13:59:38,807 [rulings_by_advance_authority.py][parser]  Failed to convert text file for: \n            04/Odisha/AAR/18-19 
